src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -24,6 +37,14 @@ function LineVis(): JSX.Element {
     return <></>;
   }
 
+  const yDomain = extendDomain(dataDomain, 0.01);
+  disableLog(yDomain[0] * yDomain[1] <= 0);
+
+  // Fallback on SymLog if Log is disabled
+  if (scaleType === ScaleType.Log && isLogDisabled) {
+    setScaleType(ScaleType.SymLog);
+  }
```

Comment:
This is needed as `scaleType` is persistent between datasets. Meaning me might switch from a dataset where the log is supported to one where it is one. In this case, the `scaleType` must not stay `Log`.